### PR TITLE
Documentation page

### PR DESCRIPTION
**Pull request by sodabarauf**
in this pull request i did these things:
-created new repository for project.
-created development branch and feature branch.
-flow GitHub workflow
-added index.html and style.css files
-setup linters.
-added descriptive README.md file.
-added nav tag and main tag into index.html file.
-added header and ul tags into nav tags.
-added section tags with header tangs and p tags in the main tag.
-added some properties into index.css file.

### CODE REVIEW

**Review by ahzamir — APPROVED** [review]
### Hi @sodabarauf 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

- Check the comments below for your `README.md` file.
# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WowOhMyGodGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Documentation Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️ Your Code is Clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: README.md [hunk 1]
```diff
@@ -1,2 +1,124 @@
-# technical-documentation-page
-###this repository is about making documentation page.
+# technical-documentation-page 🚀
+
+## Project Description 📝
+
+> this project is about making technical documentation page. it has showed HTML coding language, is uesful for web development students, included some specific information about HTML.
+
+Examples:
```
Comment (ahzamir) [inline README.md]: - Remove this `Example` it is just I write for you to know the example.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,124 @@
...
+        <li><a class="nav-link" href="#HTML_Basic_Examples">HTML Basic Examples</a>
+        </li>
+        <li><a class="nav-link" href="#HTML_Styles">HTML Styles</a>
+        </li>
+        <li><a class="nav-link" href="#HTML_Lists">HTML Lists</a>
+        </li>
+        <li><a class="nav-link" href="#HTML_Atrributes">HTML Atrributes</a>
+        </li>
+      </ul>
+    </nav>Include code snippets or examples to demonstrate your project. -->
```
Comment (ahzamir) [inline README.md]: - Remove the `<!--  -->` its for commenting the html code I shared here a text so it was like a comment but you shared an actual code so remove this.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,124 @@
...
+```css
+/* body {
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 20px;
+  background-image: url(images1.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+}You can also include CSS code snippets. */
```
Comment (ahzamir) [inline README.md]: - Remove the `/*  */` its for commenting the CSS code I shared here a text so it was like a comment but you shared an actual code so remove this.

Comment (ahzamir) [inline README.md]: - Also remove this text i the end `You can also include CSS code snippets.`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,124 @@
...
+```
+
+
+## Demo 📸
+
+Include a demo or animated GIF of your project. You can use tools like [asciinema](https://asciinema.org/) to record your terminal sessions.
+
+![Uploading documentation shot.jpg…](demo)
```
Comment (ahzamir) [inline README.md]: - This one is not working, it's not showing your project image here. Try to fix it.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,124 @@
...
+
+## Technologies Used 🛠️
+
+List the technologies or tools you used to develop your project. You can also include the libraries or frameworks you used.
```
Comment (ahzamir) [inline README.md]: - Remove this line this was just for instructing you.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,124 @@
...
+
+Examples:
```
Comment (ahzamir) [inline README.md]: - Remove this `Example`

File: README.md [hunk 7]
```diff
@@ -1,2 +1,124 @@
...
+```bash
+# git add .
+```
+
+```bash
+# git commit -m ""
+
+```bash
+# git push
+```
```
Comment (ahzamir) [inline README.md]: - Remove these as users don't push code for your project.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,124 @@
...
+## Usage 🎯
+
+first clone the repository with your local machin, then open in vscode by (code .) command, after that commit regularly from every changes that you make.
+
+```bash
+# git clone
+```
+
+```bash
+# git code .
```
Comment (ahzamir) [inline README.md]: - Remove `git code .`  change it as it is only `code .`

File: README.md [hunk 9]
```diff
@@ -1,2 +1,124 @@
...
+```
+## Features ⭐
+
+functionalities.
+
+Examples:
```
Comment (ahzamir) [inline README.md]: - Remove these two lines.

**Comment by sodabarauf** [comment]
@ahzamir 
sir, I appreciate your struggles in this part.
thanks, from your feedback, I will do my best to implement your feedback in the future projects.

**Comment by ahzamir** [comment]
> @ahzamir sir, I appreciate your struggles in this part. thanks, from your feedback, I will do my best to implement your feedback in the future projects.

@sodabarauf 
- Well done sodaba, although I already approved your project but try to implement the changes I requested for your README.md as you will share these projects later while you are job searching.

**Comment by sodabarauf** [comment]
@ahzamir 
Tnx sir, I implemented the feedback, and will try to be care about these feedbacks as well.

**Pull request merged**